sql/telemetry_derived/deviations_v1/query.sql

```diff
@@ -5,7 +5,7 @@ SELECT
   geography,
   metric
 FROM
-  `moz-fx-data-shared-prod.analysis.deviations`
+  `moz-fx-data-shared-prod.telemetry_derived.deviations_v1`
```

Comment:
This query is referencing the destination table. Do we need to add a new table name? What's the flow now?

Reply:
The flow now is `clients_first_seen` gets populated -> the anomaly detection pipeline get triggered and forecasts data for a specific date -> data gets written to `deviations_v1` -> this query gets executed to publish data to public BigQuery and JSON endpoint

Reply:
I guess it would be possible to let the anomdtct pipeline write results to a different table as it was before. But we'd end up having 2 tables with exactly the same data. Running this query with replacing the data worked for me when I tried it locally.

Reply:
This is making sense to me now. But it's still a very weird situation. This query file should include commentary to indicate this table is populated from an external job, and then this query exists only to ensure we aren't accidentally publishing data we didn't intend.

Reply:
Actually, maybe this doesn't make sense. The query puts the data in the public project. Don't we have an assumption that the pipeline will create a view named `moz-fx-data-shared-prod.telemetry_derived.deviations_v1` that points to the public table?

Reply:
Oh right, so I guess we'll want to let the anomdtct pipeline write data into a different table?

Reply:
Yes, I think we'll need to provision a different table. Perhaps `deviations_anomdtct_v1`.